internal/pkg/template/workload.go

```diff
@@ -330,6 +348,7 @@ type ParseRequestDrivenWebServiceInput struct {
 	EnableHealthCheck   bool
 	EnvControllerLambda string
 	Publish             *PublishOpts
+	Platform            *RuntimePlatformOpts
```

Comment:
What do you think about making `Platform` not a pointer? this way you don't have to check if `data.Platform == nil` each time since OS and Family will be always initialized at least to the empty string?